### PR TITLE
Generate latest from spec version 1.72.0

### PR DESCRIPTION
**Pull request by bradymholt**
This generates the client from spec version 1.72.0.

Additions:
- Adds support for creating scheduled transactions
- Adds support for retrieving transactions for a particular month

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -12,8 +12,6 @@ end
 
 task :default => [:spec]
 
-REQUIRED_RUBY_VERSION = ">= 3.3"
```
Comment (bradymholt) [inline Rakefile:15]: Moved some config to `config.yaml` to centralize it.

**Comment by bradymholt** [comment]
@dbertram - I regenerated off of 1.72.0 and pushed up changes.

File: README.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ The following methods are available in this library.
 | **Transactions**           | [transactions.get_transactions(budget_id)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/TransactionsApi.md#get_transactions)                                                                            | Returns budget transactions                           |
 |                            | [transactions.get_transactions_by_account(budget_id, account_id)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/TransactionsApi.md#get_transactions_by_account)                                          | Returns all transactions for a specified account      |
 |                            | [transactions.get_transactions_by_category(budget_id, category_id)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/TransactionsApi.md#get_transactions_by_category)                                       | Returns all transactions for a specified category     |
+|                            | [transactions.get_transactions_by_month(budget_id, month)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/TransactionsApi.md#get_transactions_by_month)                                       | Returns all transactions for a specified month     |
```
Comment (dbertram) [inline README.md:71]:
```suggestion
|                            | [transactions.get_transactions_by_month(budget_id, month)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/TransactionsApi.md#get_transactions_by_month)                                                   | Returns all transactions for a specified month     |
```

File: README.md [hunk 3]
```diff
@@ -77,6 +78,7 @@ The following methods are available in this library.
...
 | **Scheduled Transactions** | [scheduled_transactions.get_scheduled_transactions(budget_id)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/ScheduledTransactionsApi.md#get_scheduled_transactions)                                     | Returns all scheduled transactions                    |
 |                            | [scheduled_transactions.get_scheduled_transaction_by_id(budget_id, scheduled_transaction_id)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/ScheduledTransactionsApi.md#get_scheduled_transaction_by_id) | Returns a single scheduled transaction                |
+|                            | [scheduled_transactions.create_scheduled_transaction(budget_id, data)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/ScheduledTransactionsApi.md#create_scheduled_transaction)                                                                  | Creates a single scheduled transaction                      |
```
Comment (dbertram) [inline README.md:81]:
```suggestion
|                            | [scheduled_transactions.create_scheduled_transaction(budget_id, data)](https://github.com/ynab/ynab-sdk-ruby/blob/master/docs/ScheduledTransactionsApi.md#create_scheduled_transaction)                           | Creates a single scheduled transaction                      |
```

File: docs/ScheduledTransactionFrequency.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# YNAB::ScheduledTransactionFrequency
+
+## Properties
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
+
```
Comment (dbertram) [inline docs/ScheduledTransactionFrequency.md:7]: Does the ruby generator not output the enum values?

File: config.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+gemName: ynab
+moduleName: YNAB
+gemSummary: YNAB API Client for Ruby
+gemDescription: Ruby gem wrapper for the YNAB API.
```
Comment (dbertram) [inline config.yaml:4]:
```suggestion
gemDescription: Ruby gem wrapper for the YNAB API. API documentation available at https://api.ynab.com.
```
?

**Review by dbertram — APPROVED** [review]
A couple formatting nits and a couple questions, but otherwise looks great!

File: docs/ScheduledTransactionFrequency.md (hunk 4 above)
Comment (bradymholt) [reply docs/ScheduledTransactionFrequency.md:7]: No, unfortunately not.  I believe it's not supported in OpenAPI Generator for Ruby (yet).

Comment (dbertram) [reply docs/ScheduledTransactionFrequency.md:7]: :whomp: :whomp:

**Pull request merged**